### DISCUSSION TITLE
Content Management Overview | FixIt

### DISCUSSION THREAD

**Discussion by giscus** [hugo-fixit/FixIt#380] [General]
# Content Management Overview | FixIt

Find out how to create and organize your content quickly and intuitively in FixIt theme.

https://fixit.lruihao.cn/docs/content-management/introduction/

**Comment by ZarkMedo** [comment]
cool

**Comment by niksihora** [comment]
Hi,
I'm Unable to set featured image in home page.
It is visible on individual post level but not on homepage, Is there any variable which needs to e enabled ?

**Reply by Lruihao** [reply]
Perhaps you can refer to the cover image settings of this article. https://github.com/hugo-fixit/docs/tree/main/content/en/documentation/installation

**Reply by niksihora** [reply]
I have published a site using FixIt theme @https://itbaba.org/
Source code is available @https://github.com/niksihora/niksihora.github.io.
I'm using below in index file as well. On individual post I can see the featured image but it is not visible from home page.
resources:
  - name: featured-image
    src: kubernetes.png

**Reply by Lruihao** [reply]
OK, I got it, I'll try tomorrow.

**Reply by Lruihao** [reply]
I just update your theme to the latest, It's work!
<img width="1397" alt="image" src="https://github.com/hugo-fixit/FixIt/assets/33419593/7c20df31-f3a8-49d0-8b11-b5a22dd1ac30">

**Reply by Lruihao** [reply]
@niksihora I push the commit for your blog repo in https://github.com/niksihora/niksihora.github.io/pull/1

**Reply by niksihora** [reply]
Thank you 😊, Working great now.
